### ISSUE TITLE
3DS2 Android not working with some cards:  Invalid JSON for NativeChallenge. No value for challengeInfoLabel

### ISSUE THREAD

**Issue by Gp2mv3** [Adyen/adyen-react-native#315]
**Describe the bug**
When we try to authenticate some cards (Some Visa Keytrade Bank for example), we receive the following error:

```
{"description": "[Ljava.lang.StackTraceElement;@4e60a37", "errorCode": "unknown", "message": "Protocol Error - 
transactionID: cb6a3a8d-5b7c-4873-9893-33f52db8d2c5
errorCode: 201
errorDescription: Required Data Element Missing.
errorDetails: Invalid JSON for NativeChallenge. No value for challengeInfoLabel"}
```

Here is the request sent by the app:

```
{
  returnUrl: 'adyencheckout://com.sparks',
  storePaymentMethod: true,
  amount: { value: 0, currency: 'EUR' },
  paymentMethod: {
    brand: 'visa',
    threeDS2SdkVersion: '2.2.13',
    holderName: 'Hadrien crespin',
    encryptedCardNumber: 'adyenan0_1_1$cfeXn...nx0MFF',
    encryptedSecurityCode: 'adyenan0_1_1$qP99L...NQC0=',
    encryptedExpiryMonth: 'adyenan0_1_1$ddB7...MAU=',
    encryptedExpiryYear: 'adyenan0_1_1$eT+2PGJe...USQ==',
    type: 'scheme',
    checkoutAttemptId: '080f0e49-055...9A17763107DD60'
  },
  reference: 'preAuth-user-6555f44ad59a4065ca60aeb6',
  additionalData: {
    allow3DS2: true,
    executeThreeD: true,
    authorisationType: 'PreAuth',
    shopperEmail: '...'
  },
  channel: 'Android',
  shopperReference: 64ac343143afb3ff0016c14d,
  shopperEmail: '...',
  shopperIP: '::ffff:127.0.0.1',
  shopperInteraction: 'Ecommerce',
  recurringProcessingModel: 'UnscheduledCardOnFile',
  shopperStatement: '...',
  merchantAccount: '...',
  authenticationData: { threeDSRequestData: { nativeThreeDS: 'preferred' } }
}
```

Here is the challenge response:
```
{
  "action": {
    "paymentData": "AbL...aE=",
    "paymentMethodType": "scheme",
    "authorisationToken": "Ab02b....wmw",
    "subtype": "fingerprint",
    "token": "eyJk...GMifQ==",
    "type": "threeDS2"
  },
  "resultCode": "IdentifyShopper"
}
```

**To Reproduce**
1. Try a payment with native 3Ds2
2. You'll receive the error after the answer from /payment/payments

**Expected behavior**
Normally a 3Ds authentication should be asked to the user.

**Desktop (please complete the following information):**
 - OS: Android
 - Version 1.1.0

**Smartphone (please complete the following information):**
 - Device: Pixel 6 (Also seen on other Android phones)
 - OS: 14

**Additional context**
The Authentication doesn't work either on iOS. Also the transaction stays in "Offer" and not in "Transactions" when looking at the Adyen interface.

**Comment by Gp2mv3** [comment]
Seems to come from the Issuer's side ? https://github.com/Adyen/adyen-3ds2-android/issues/26

**Comment by Gp2mv3** [comment]
On version 1.2.0 it seems to work: The 3Ds2 authentication is not asked, at least on our previously not working card.
Maybe not the best solution for security but at least users are able to enter their credit card.

**Comment by descorp** [comment]
Hey @Gp2mv3 

This doesn't sound like a scope of the React Native SDK 😅
Could you reach out to [Adyen Support](https://www.adyen.help/hc/en-us/requests/new?ticket_form_id=360000705420)?

**Comment by descorp** [comment]
Hey @Gp2mv3 

Any luck?

**Comment by Gp2mv3** [comment]
Hello,

I contacted the support at the same time as writing the ticket.
Unfortunately, no solution was found by them neither. The issue seems to come from a bad implementation of the 3DS2 protocol by the issuer.

On our side we upgraded the react-native-adyen package and the backend implementation to be on the last API versions. 

We also disabled the "always" for the 3DSauthentication, and use the "preferred" instead. That "solves" the issue until the issuer correct the implementation.

So indeed it doesn't come from the Adyen's side and neither from the RN SDK.
